### ISSUE TITLE
enum: initial assignment

### ISSUE THREAD

**Issue by markus2330** [ElektraInitiative/libelektra#521]
It seems like that initially (no key in backend -> at least one key in backend) enum does not validate instead every value is accepted.

```
$ kdb rm -r user/example/highlevel/
$ kdb set /example/highlevel/device smartphones
Using name user/example/highlevel/device
Create a new key user/example/highlevel/device with string smartphones
$ kdb set /example/highlevel/device smartphones
Using name user/example/highlevel/device
Set string to smartphones
The command set failed while accessing the key database with the info:
Error (#121) occurred!
Description: Validation failed
Ingroup: plugin
Module: enum
At: /home/jenkins/workspace/workspace/elektra-git-buildpackage-jessie/libelektra/src/plugins/enum/enum.c:102
Reason: Validation failed
Mountpoint: user/example/highlevel
Configfile: /home/markus/.config/highlevel.ini.8594:1455571273.913728.tmp
$ kdb set /example/highlevel/device smartphone
Using name user/example/highlevel/device
Set string to smartphone
```

The problem might be in spec. I use
```
$ kdb mount highlevel.ini spec/example/highlevel ni 
$ kdb spec-mount /example/highlevel -v
```

**Comment by tom-wa** [comment]
@markus2330 fixed in #540

**Comment by markus2330** [comment]
Thank you!
